src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs

```diff
@@ -38,5 +40,7 @@
      [react/view {:style {:height 1 :background-color "#e8ebec" :margin-horizontal 16}}]
      [react/touchable-highlight {:on-press #(re-frame/dispatch [:logout])
                                  :style {:margin-top 60}}
-      [react/view
-       [react/text {:style {:color :red}} "Log out"]]]]))
+
+      [react/view {:style styles/logout-row}
+       [react/text {:style {:color :red}} "Log out"]
```

Comment:
Maybe it makes sense to use colors from https://github.com/status-im/status-react/blob/develop/src/status_im/ui/components/colors.cljs, as the shades for red are gray are different there.

Reply:
nice catch! alright will pull those in. thanks @siphiuel